commands/azureCommands/acr-build-logs.ts

```diff
@@ -0,0 +1,381 @@
+        const osType: string = log.platform.osType ? log.platform.osType : '?';
+        const name: string = log.name ? log.name : '?';
+        let imageOutput: string = '';
+
+        let needsNA: boolean = false;
+        if (logData.logs[i].outputImages) {
+            for (const img of log.outputImages) {
+                if (img) {
+                    const tag: string = img.tag ? img.tag : '?';
+                    const repository: string = img.repository ? img.repository : '?';
+                    const registry: string = img.registry ? img.registry : '?';
+                    const digest: string = img.digest ? img.digest : '?';
+                    imageOutput += `<tr>
+                                    <td>${tag}</td>
+                                    <td>${repository}</td>
+                                    <td>${registry}</td>
+                                    <td>${digest}</td>
+                                </tr>`;
+                }
+            }
+            if (!log.outputImages[0]) {
+                needsNA = true;
+            }
+        } else {
+            needsNA = true;
+        }
+        if (needsNA) {
+            imageOutput += `<tr>
+                                <td>NA</td>
+                                <td>NA</td>
+                                <td>NA</td>
+                                <td>NA</td>
+                            </tr>`;
+        }
+
+        panel.webview.postMessage({
+            'type': 'populate',
+            'id': i,
+            'logComponent': `
+                    <div class = "holder">
+                        <button id= "btn${i}" class="accordion">
+                            <table>
+                                <tr>
+                                    <td class = 'widthControl'>${name}</td>
+                                    <td class = 'widthControl'>${buildTask}</td>
+                                    <td class ='status widthControl ${log.status}'>${log.status}</td>
+                                    <td class = 'widthControl'>${startTime}</td>
+                                    <td class = 'widthControl'>${finishTime}</td>
+                                    <td class = 'widthControl'>${osType}</td>
+                                    <td><div class = "arrow"></div></td>
+                                </tr>
+                            </table>
+                        </button>
+                        <div class="panel">
+                            <table class="overallTable">
+                                <tr>
+                                    <td colspan="4">Output Images</td>
+                                </tr>
+                                <tr>
+                                    <td>Tag</th>
+                                    <td>Repository</td>
+                                    <td>Registry</td>
+                                    <td>Digest</td>
+                                </tr>
```

Comment:
Increase bottom padding of Output Images box. It's too close to the next build log under and unpleasing to the eye